packages/shared/src/utils/date/date-time.ts

```diff
@@ -13,7 +13,8 @@ type TExtendedMoment = typeof moment & {
 
 // Localize moment instance with specific object
 export const initMoment = (lang: string) => {
-    if (!lang || lang === 'EN') return moment;
+    const ignored_language = ['EN', 'BN'];
```

Comment:
why do we ignore BN?

Reply:
We ignoring BN because the translation is also happening towards the date and time. In DTrader and also POI when we submit the date, it cannot be recognise by BE. So we make BN date to be in EN format to solve this issue